doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_ckan_sink.md

```diff
@@ -50,32 +50,58 @@ This is done at the cygnus-ngsi Http listeners (in Flume jergon, sources) thanks
+|---|---|---|
+| `/` | `<fiware-service>` | `<fiware-service>` | 
 
 [Top](#top)
 
 #### <a name="section1.2.2"></a>Packages/datasets naming conventions
-A package/dataset named as the concatenation of the notified `fiware-service` and `fiware-servicePath` header values (or, in absence of such headers, the defaulted value for the FIWARE service and service path) is created (if not existing yet) in the above organization.
+* Data model by entity (`data_model=dm-by-entity`). A package/dataset named as the concatenation of the notified `fiware-service` and `fiware-servicePath` header values (or, in absence of such headers, the defaulted value for the FIWARE service and service path) is created (if not existing yet) in the above organization.
+Since based in [PostgreSQL only accepts](https://www.postgresql.org/docs/current/static/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS), it must be said only alphanumeric characters and the underscore (`_`) are accepted. The hyphen ('-') is also accepted. This leads to  certain [encoding](#section2.3.3) is applied depending on the `enable_encoding` configuration parameter
+Nevertheless, different than PostgreSQL, [dataset lengths](http://docs.ckan.org/en/latest/api/#ckan.logic.action.create.package_create) may be up to 100 characters (minimum, 2 characters).
 
-Since based in [PostgreSQL only accepts](https://www.postgresql.org/docs/current/static/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS), it must be said only alphanumeric characters and the underscore (`_`) are accepted. The hyphen ('-') is also accepted. This leads to  certain [encoding](#section2.3.3) is applied depending on the `enable_encoding` configuration parameter.
 
-Nevertheless, different than PostgreSQL, [dataset lengths](http://docs.ckan.org/en/latest/api/#ckan.logic.action.create.package_create) may be up to 100 characters (minimum, 2 characters).
+* Data model by entity id (`data_model=dm-by-entity-id`). A package/dataset name always take the entity ID. Such a name is already given in the NGSIEvent values, see the [Configuration](#section2.1) section for more details) within the the `NGSIEvent`.
+
+The following table summarizes the package name composition:
+
+| FIWARE service path | `dm-by-entity` | `dm-by-entity-id` |
+|---|---|---|
+| `/` | `<fiware-service>_<fiware-servicePath>` | `<entityId>` | 
 
 [Top](#top)
 
 #### <a name="section1.2.3"></a>Resources naming conventions
-CKAN resources follow a single data model (see the [Configuration](#section2.1) section for more details), i.e. per entity. Thus, a resource name always take the concatenation of the entity ID and type. Such a name is already given in the `notified_entities`/`grouped_entities` header values (depending on using or not the grouping rules, see the [Configuration](#section2.1) section for more details) within the `NGSIEvent`.
+The resiurce name depends on the configured data model (see the [Configuration](#section2.1) section for more details):
```

Comment:
```suggestion
The resource name depends on the configured data model (see the [Configuration](#section2.1) section for more details):
```

Reply:
Fixed in [db4c369](https://github.com/telefonicaid/fiware-cygnus/pull/1850/commits/db4c369038e851be614e0c5e15585f31d3a1303a)